src/mongo_logger.ts

```diff
@@ -732,12 +740,40 @@ export class MongoLogger {
     this.componentSeverities = options.componentSeverities;
     this.maxDocumentLength = options.maxDocumentLength;
     this.logDestination = options.logDestination;
+    this.pendingLog = undefined;
   }
 
   willLog(severity: SeverityLevel, component: MongoLoggableComponent): boolean {
     return compareSeverity(severity, this.componentSeverities[component]) <= 0;
   }
 
+  turnOffSeverities() {
+    for (const key of Object.keys(MongoLoggableComponent)) {
+      this.componentSeverities[key as MongoLoggableComponent] = SeverityLevel.OFF;
+    }
+  }
+
+  private logWriteFailureHandler(error: Error) {
+    try {
+      const tempLogDestination = createStdioLogger(process.stderr);
```

Comment:
I think we want a condition here, if the logger was already using stderr then don't do this.